hoomd/hpmc/pytest/test_shape.py

```diff
@@ -3,40 +3,41 @@
 from hoomd.hpmc import _hpmc
 import numpy as np
 import pytest
+from hoomd.hpmc.pytest.conftest import *
+
+
+def check_dict(shape_dict, args):
+    for key, val in args.items():
```

Comment:
A developer docstring as to the function that `check_dict` does would be helpful.